charts/opensearch/CHANGELOG.md

```diff
@@ -227,7 +236,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Security
 
 
-[Unreleased]: https://github.com/opensearch-project/helm-charts/compare/opensearch-2.11.1...HEAD
+[Unreleased]: https://github.com/opensearch-project/helm-charts/compare/opensearch-2.11.5...HEAD
+[2.11.5]: https://github.com/opensearch-project/helm-charts/compare/opensearch-2.11.4...opensearch-2.11.5
```

Comment:
Any idea why we are missing `2.11.2` to `2.11.4`?

Reply:
They are missing from old PR's @zelinh

Reply:
Updated in my latest commit, please check.